config/test.js

```diff
@@ -32,7 +32,7 @@ module.exports = {
   },
 
   cache: {
-    volumeCachePath: path.join(__dirname, '..', 'test', 'tmp')
+    volumeCachePath: path.join('/tmp', 'test-cache')
```

Comment:
This was me trying to fix failing tests that really shouldn't have been. I'll take a closer look at this patch later.